metrotranscom/doorway
# feat: new limited jurisdictional admin role

This PR addresses [#(490)](https://app.zenhub.com/workspaces/bloom-5dc32d7144bd400001315dac/issues/gh/metrotranscom/doorway/490)


## Description
New limited jurisdictional admin role for Doorway. This role should have all the same permissions as an standard jurisdictional admin, but without access to any PII. This means they CANNOT do the following: see the partners users tab, see/edit/delete partner or public users, see/edit/delete listing applications, submit paper applications, see/resolve/process application flagged sets (aka application duplicates). This should be across the partners site and the API.

## How Can This Be Tested/Reviewed?

Test a user with the new role can be created. To do so, login as an admin user to the partners portal, navigate to the users tab, add a new user and select Jurisdictional Admin - No PII.

Use seeded or newly created limited jurisdictional admin to test all listing and settings flows successfully. Check that the user cannot do anything of the actions outlined in the description.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
